### PR TITLE
First patch for CPO to test lb on catalyst

### PR DESCRIPTION
**Pull request by bzhaoopenstack**
1. Add catalyst openrc
2. Add the related dockerhub configuration.
3. Add a test script into tools dir for downloading and executing on test node.

### CODE REVIEW

File: tools/test-lb-service-catalyst-cloud.sh [hunk 1]
```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
```
Comment (wangxiyuan) [inline tools/test-lb-service-catalyst-cloud.sh]: this is only used for CPO job, no need to add it here. Just move to the job folder.

Comment (bzhaoopenstack) [reply tools/test-lb-service-catalyst-cloud.sh]: OK

File: roles/export-cloud-openrc/tasks/main.yaml [hunk 2]
```diff
@@ -131,6 +131,22 @@
...
+  set_fact:
+    openrc:
+      OS_PASSWORD: '{{ catalyst_credentials.password }}'
+      OS_AUTH_URL: '{{ catalyst_credentials.auth_url }}'
+      OS_IDENTITY_API_VERSION: '{{ catalyst_credentials.identity_api_version }}'
+      OS_PROJECT_DOMAIN_NAME: '{{ catalyst_credentials.project_domain_name }}'
+      OS_PROJECT_NAME: '{{ catalyst_credentials.project_name}}'
+      OS_PROJECT_ID: '{{ catalyst_credentials.project_id}}'
+      OS_REGION_NAME: '{{ catalyst_credentials.region_name}}'
+      OS_TENANT_NAME: '{{ catalyst_credentials.tenant_name }}'
```
Comment (liusheng) [inline roles/export-cloud-openrc/tasks/main.yaml]: dont need `tenant_name` and delete it in credentials secrets, juse use `catalyst_credentials.project_name`

File: zuul.d/secrets.yaml [hunk 3]
```diff
@@ -911,6 +911,186 @@
           Jkqk6acsQd1x5vRFGW5AZsOdiLTD7EOSHbvusEmF+V3/n9tGWABc+w6pz0n8xp5+V/SEv
           GAlr1E9/fPkPOlEJSh87XhH8srX9Hh3ZFtuAYXeDwuhv0yf0WifDOz3Sz+kJAA=
 
+- secret:
+    name: catalyst_credentials
+    data:
+      auth_type: password
```
[the remark this answers is not in the document]
Comment (bzhaoopenstack) [reply zuul.d/secrets.yaml]: fix

File: roles/export-cloud-openrc/tasks/main.yaml (hunk 2 above)
Comment (bzhaoopenstack) [reply roles/export-cloud-openrc/tasks/main.yaml]: fix

**Pull request merged**